cognite/client/config.py

```diff
@@ -41,7 +41,7 @@ def __init__(self) -> None:
         self.max_connection_pool_size: int = 50
         self.disable_ssl: bool = False
         self.proxies: dict[str, str] | None = {}
-        self.max_workers: int = 10
+        self.max_workers: int = 5
```

Comment:
Worth adding a note to changelog?

Reply:
Yes, I think so. if someone rely on it, it would be good for them to go back in the changelog to figure out what happened.